importer/api/__init__.py

```diff
@@ -151,6 +154,23 @@ def import_children(course, element, parent):
             )
             for asset in assets:
                 resource.static_assets.add(asset)
+    else:
+        soup = BeautifulSoup(etree.tostring(element), 'lxml')
+        for child in soup.findAll():
+            for _, val in child.attrs.items():
+                try:
+                    if val.startswith(target):
+                        path = val[len(target):]
+                        try:
+                            asset = StaticAsset.objects.get(
+                                course__id=resource.course_id,
+                                asset=course_asset_basepath(course, path),
+                            )
+                            resource.static_assets.add(asset)
+                        except StaticAsset.DoesNotExist:
+                            continue
+                except AttributeError:
+                    continue  # not a string
```

Comment:
Could you please explain briefly at the beginning of this section what is the logic you implement?